docs/development/api_updates.md

```diff
@@ -3,7 +3,7 @@
 Kops uses the Kubernetes API machinery.  It is well designed, and very powerful, but you have to
 jump through some hoops to use it.
 
-Recommended reading: [kubernetes API changes doc](https://github.com/kubernetes/community/blob/master/contributors/devel/api_changes.md)
+Recommended reading: [kubernetes API convention doc](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md)
```

Comment:
The api_changes.md still exists under sig-architecture as well:

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api_changes.md

I think both are useful to understand, so maybe we include both links?

Reply:
You're right. I will add both links.